nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -100,6 +100,8 @@ The module update takes care of the new config syntax and the data itself (user
 
 - `programs.gnupg.agent.pinentryFlavor` is now set in `/etc/gnupg/gpg-agent.conf`, and will no longer take precedence over a `pinentry-program` set in `~/.gnupg/gpg-agent.conf`.
 
+- The `osquery` package and an an experimental `services.osquery` module have been re-added.
```

Comment:
It's a new service, not just a package, please move it to "New Services" and drop the mention of the package, it's redundant.